th2_data_services_lwdp/utils/misc.py

```diff
@@ -1,18 +1,47 @@
 from datetime import datetime, timezone
+from th2_data_services.interfaces.utils.converter import ITimestampConverter, TimestampType
+
+
+class DatetimeConverter(ITimestampConverter):
```

Comment:
I think we don't need this class
because we have https://github.com/th2-net/th2-data-services/blob/master/th2_data_services/provider/utils/converters.py#L4 
in master
let's copy it to DS core with the name "ProtobufTimestampConverter"

Reply:
after that, use this copied class instead

Reply:
Method moved, Class moved and renamed.
https://github.com/th2-net/th2-data-services/pull/212